src/js/letters/containers/VeteranBenefitSummaryLetter.jsx

```diff
@@ -3,10 +3,14 @@ import { connect } from 'react-redux';
 
 import _ from 'lodash';
 
-import { veteranBenefitSummaryOptionText } from '../utils/helpers';
 import { updateBenefitSummaryOption } from '../actions/letters';
+import {
+  characterOfServiceContent,
+  veteranBenefitSummaryOptionText
+} from '../utils/helpers';
+import { formatDateShort } from '../../common/utils/helpers';
 
-class VeteranBenefitSummaryLetter extends React.Component {
+export class VeteranBenefitSummaryLetter extends React.Component {
```

Comment:
Need to export this class in order to import the unconnected component for testing.